csharp/ql/src/experimental/Security Features/backdoor/PotentialTimeBomb.ql

```diff
@@ -0,0 +1,152 @@
+/**
```

Comment:
I think we're missing a `.qhelp` file for this one

Reply:
I forgot to create it, working on it (will check in with the other fixes)